### DISCUSSION TITLE
Missing LOGS on Dashboard anyone?

### DISCUSSION THREAD

**Discussion by anshunjain** [supabase/supabase#20813] [Questions] [answered]
For the last week, I am facing issues, where I am 100% certain that the code is running, and i can even inspect and see the code. I ran a simple method to test it on my instance right now which all it does is raise a log, and that log is not to be seen in the log browser. i searched specifically for the log text I pasted, and also for the recent logs etc. 

If this is true this is highly concerning, since without logs you cannot do any development.

**Comment by Ziinc** [comment, marked as answer]
Hi if this issue persists for your project, please open a support ticket so that we can investigate this issue further.
